charts/fint-altinn-ebevis-service/templates/deployment.yaml

```diff
@@ -28,10 +28,10 @@ spec:
               mountPath: {{ .Values.deployment.volumeMounts.mountPath}}
           resources:
             limits:
-              memory: {{ .Values.deployment.resources.limits.memory}}
+              memory: "{{ include "fint-altinn-ebevis-service.xmx" . }}Mi"
```

Comment:
change from "{{ include "fint-altinn-ebevis-service.xmx" . }}Mi" to {{ include "fint-altinn-ebevis-service.xmx" . }}Mi